test/unit/core.js

```diff
@@ -1709,3 +1709,45 @@ QUnit.test( "Iterability of jQuery objects (gh-1693)", function( assert ) {
 		assert.ok( true, "The browser doesn't support Symbols" );
 	}
 } );
+
+QUnit[ jQuery.Deferred ? "test" : "skip" ]( "jQuery.readyException (original)", function( assert ) {
+	assert.expect( 1 );
+
+	var message,
+		origWindowSetTimeout = window.setTimeout;
+
+	window.setTimeout = function( fn ) {
```

Comment:
@markelog I had to create a sandbox and use `this.sandbox.stub` instead of `sinon.stub` or stubs wouldn't have been cleaned up (because how would Sinon know when to clean up?). It works fine now.